moveit_experimental/moveit_jog_arm/include/moveit_jog_arm/boost_pool_allocation.h

```diff
@@ -36,25 +36,14 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 *******************************************************************************/
 
-// Server node for arm jogging with MoveIt.
-
 #pragma once
 
-#include "jog_interface_base.h"
+#include <boost/pool/pool_alloc.hpp>
```

Comment:
I'm not sure it even belongs in jog_arm proper and not in some utility project.  Figured it was better to separate it because it isn't really related other than that jog_arm is using it.

Reply:
I made this change (moveit::util namespace), lmk what you think?